docker-bake.hcl

```diff
@@ -23,10 +16,6 @@ variable "DALEC_DISABLE_DIFF_MERGE" {
     default = "0"
 }
 
-variable "DALEC_NO_CACHE_EXPORT" {
-    default = "0"
-}
-
 target "frontend" {
```

Comment:
Can you add a comment here clarifying that this will build the root Dockerfile (that builds the frontend image) since no other build steps are specified?

Reply:
Done.

Reply:
done